apps/pie-microsite/src/__tests__/utilities/helpers.test.js

```diff
@@ -49,4 +49,18 @@ describe('helpers.js', () => {
             });
         });
     });
+
+    describe('numberHelpers', () => {
+        describe('isNumber', () => {
+            [['1', true], ['fooBarBaz', false], [1, true]].forEach(example => {
```

Comment:
Personally I find the cases a bit easier to read when each one is put on its own line